src/vs/workbench/contrib/chat/browser/media/chat.css

```diff
@@ -1390,6 +1390,29 @@ have to be updated for changes to the rules above, or to support more deeply nes
 	height: auto;
 }
 
+/* Icon-only chat attachment button styling */
+.action-item.chat-attachment-button {
+	width: 20px;
+	height: 20px;
+}
+
+.interactive-session .action-item.chat-attachment-button .action-label {
+	display: flex;
```

Comment:
nit and not blocking: double check how much of this css is inherited, can maybe clean it up a tiny tiny bit

<img width="252" height="685" alt="Image" src="https://github.com/user-attachments/assets/792dd822-b23e-4ad9-8fa0-edd75d58ebaf" />